src/cmd/main.go

```diff
@@ -33,7 +33,7 @@ func main() {
 	info := component.BuildInfo{
 		Command:     "swi-k8s-opentelemetry-collector",
 		Description: "SolarWinds distribution for OpenTelemetry",
-		Version:     "0.11.1",
+		Version:     "0.12.0",
```

Comment:
Why not continue with 0.11.x? It shouldn't be used by any released `k8s collector` yet.

Reply:
Ok I updated image to 0.11.2.